src/keyboard/LayerPicker.tsx

```diff
@@ -172,7 +172,7 @@ export const LayerPicker = ({
         {onRemoveClicked && (
           <button
             type="button"
-            className="hover:text-primary-content hover:bg-primary rounded-sm"
+            className="hover:text-primary-content hover:bg-primary rounded-sm disabled:hover:text-primary-content disabled:hover:bg-transparent disabled:cursor-not-allowed disabled:opacity-20"
```

Comment:
`disabled:hover:text-primary-content` seems unnecessary unless I'm missing something. Also seems wrong, as it should only be used when the background is `bg-primary`

Reply:
when standard users reached the max number of layers, it seems like they can add more because the button is still active.

![Screenshot 2024-12-15 at 10 58 51](https://github.com/user-attachments/assets/d8dde3f9-275a-45bd-9ea2-9461d81b9913)

adding the disabled state conveys that a max number of layers has been reached.

![Screenshot 2024-12-15 at 11 08 32](https://github.com/user-attachments/assets/8b301ce5-9c5f-4af3-8d20-7fb38e9040e6)

this simple visual feedback reduces the amount of confusion with potential standard users.


> Also seems wrong, as it should only be used when the background is bg-primary

you are correct, a simple pointer event should not require any changes to text/bg colors. the update i made below should be a lot simplier

```diff
- className="hover:text-primary-content hover:bg-primary rounded-sm disabled:hover:text-primary-content disabled:hover:bg-transparent disabled:cursor-not-allowed disabled:opacity-20"
+ className="hover:text-primary-content hover:bg-primary rounded-sm disabled:opacity-50 disabled:pointer-events-none"
```

Reply:
Yep understood the disabled state is needed, was more concerned about the text color usage. This simpler case makes sense!